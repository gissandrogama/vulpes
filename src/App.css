@font-face{
  font-family: Industrial;
  src: url(./assests/Industry/Industry-Bold.ttf)
}
*{
  margin: 0;
  padding: 0;
  outline: 0;
  box-sizing: border-box;    
}

html, body{
  height: 100%;
}

#root {
  min-height: 100%;
  position: relative;
}

body {
  background: #000 url('./assests/background2.png') no-repeat;
  background-size: cover;
  -webkit-font-smoothing: antialiased !important;
}

body, input, button{
  font-family: Industrial, Arial, Helvetica, sans-serif;
  font-size: 14px;
}

.container {
  width: 100%;
  height: 100%;
  display: flexbox; 
}

.header {
  position: relative;
  width: 100%;
  justify-content: center; 
  align-items: center;
  align-content: center; 
  display: flex;
  height: 95px;
}

.logo {
  height: 100%;
  margin: 0 30px 0 0;
}

.logo img {
  height: 100%;
}

.menu {
  height: 100%;
}

.menulist {
  list-style-type: none;
  padding: 10px;  
  margin: 20px auto;
}

.menulist_item {
  display: inline-block;
  height: 100%;
  margin-right: 10px;
}

.menulist_item:last-child {
  margin-right: 0;
}

.menulink {
  color: #FFF;
  display: inline-block;
  height: 100%;
  font-size: 18px;
  text-decoration: none;
  text-transform: uppercase;
  display: flex;
  align-items: center;
  padding: 0 30px;
}

.menulink:hover {
  color: #f54900;
}

.menulink--active {
  color: #f54900;    
  font-weight: 800;
}

.direita {
  position: absolute;  
  width: 40%;
  height: 85.8%; 
  bottom: 0;
  right: 0;                 
}

.direita img {
  height: 100%;
  width: 400px;
  float: right;    
}

.esquerda {
  width: 50%;
  background-color: #FFF;      
}

.esquerda h1 {
  font-size: 30px;
  color: #f54900;
  letter-spacing: 1rem;  
}