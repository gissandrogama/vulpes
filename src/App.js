import React from 'react';
import './App.css';

function App() {
  return (
    <h1>Vulpes</h1>
  );
}

export default App;
