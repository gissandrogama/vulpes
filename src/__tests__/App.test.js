import React from 'react'
import { shallow } from 'enzyme'
// import renderer from 'react-test-renderer';
import App from '../App'

describe('Testing App component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<App />)

    expect(wrapper.html()).toMatchSnapshot();
  });
});