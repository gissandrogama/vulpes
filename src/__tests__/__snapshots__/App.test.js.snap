// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App component should render correctly 1`] = `"<div class=\\"header\\"><div class=\\"logo\\"><img src=\\"logo3.png\\" alt=\\"Vulpes\\"/></div><div class=\\"menu-desk\\"><ul class=\\"menulist\\"><li class=\\"menulist_item\\"><a href=\\"/\\" class=\\"menulink menulink--active\\">HOME</a></li><li class=\\"menulist_item\\"><a href=\\"/loja\\" class=\\"menulink\\">LOJA</a></li><li class=\\"menulist_item\\"><a href=\\"#\\" class=\\"menulink\\">EQUIPES</a></li><li class=\\"menulist_item\\"><a href=\\"#\\" class=\\"menulink\\">PARCEIROS</a></li><li class=\\"menulist_item\\"><a href=\\"/sobre\\" class=\\"menulink\\">SOBRE</a></li></ul></div></div><div class=\\"container\\"><div class=\\"content\\"><h1>VULPES</h1><p>Olá, nós somos um time</p><p>competitivo de eSport.</p><button class=\\"btn\\">VER MAIS</button></div><div class=\\"direita\\"><img src=\\"avatar.png\\" alt=\\"Avatar\\"/></div></div>"`;
